GitCommands/Git/GitModule.cs

```diff
@@ -3347,10 +3347,11 @@ public string GetFileText(ObjectId id, Encoding encoding)
         [CanBeNull]
         public ObjectId GetFileBlobHash(string fileName, ObjectId objectId)
         {
-            if (objectId == ObjectId.WorkTreeId)
+            if (objectId == ObjectId.WorkTreeId || objectId == ObjectId.CombinedDiffId)
             {
                 // working directory changes
-                Debug.Assert(false, "Tried to get blob for worktree file");
+                Debug.Assert(false, "Tried to get blob for unsupported revision: " +
```

Comment:
[nit] Interpolated it would probably read nicer

Reply:
Done. Also made into ArgumentException, reviewed use again and ignored Combined in one call path (where it never was used though).